src/TestSuite/IntegrationTestCase.php

```diff
@@ -913,14 +913,15 @@ public function assertResponseEquals($content, $message = '')
      *
      * @param string $content The content to check for.
      * @param string $message The failure message that will be appended to the generated message.
+     * @param bool   $ignoreCase A flag to check whether we should ignore case or not.
      * @return void
      */
-    public function assertResponseContains($content, $message = '')
+    public function assertResponseContains($content, $ignoreCase = false, $message = '')
     {
```

Comment:
Then I would have to always specify the message parameter also. But I agree that we need it to be backward compatible. I'll amend it.